projects/frontend/src/features/Home.jsx

```diff
@@ -1,9 +1,26 @@
-import { Link } from 'react-router-dom'
+import { useEffect, useState } from 'react'
+import { getHealth } from '../services/health'
+
+const API_URL = import.meta.env.VITE_API_URL
 
 export const Home = () => {
+  const [health, setHealth] = useState()
+
+  useEffect(() => {
+   getHealth().then((response) => setHealth(response))
```

Comment:
Should we add `setHealth` to the `useEffect` array?

Reply:
We can do that but in general, the `setState` is stable, it never changes. This means that we can omit it from the `useEffect` or also `useCallback` dependency list.